src/django/sqls/remove_items_without_matches_and_related_facilities.sql

```diff
@@ -0,0 +1,59 @@
+CREATE OR REPLACE PROCEDURE remove_items_without_matches_and_related_facilities()
+LANGUAGE plpgsql
+AS $$
+BEGIN
+    -- Create temporary tables to store the IDs of the items that do not have matches and are not have facilities created from them
+    CREATE TEMPORARY TABLE temporary_facility_list_item_ids AS
+    SELECT id 
+    FROM api_facilitylistitem 
+        WHERE NOT EXISTS (
+        SELECT 1
+        FROM api_facilitymatch
+        WHERE api_facilitymatch.facility_list_item_id = api_facilitylistitem.id
+    )
+    AND NOT EXISTS (
+        SELECT 1
+        FROM api_facility
+        WHERE api_facility.created_from_id = api_facilitylistitem.id
+    );
+
+    CREATE TEMPORARY TABLE temporary_facility_list_item_temp_ids AS
```

Comment:
Why do we need temp table, could we just store it in variable to use?

Reply:
+1 temp table is quite heavy operation, please read the thread: https://stackoverflow.com/questions/73665662/when-is-it-better-to-use-cte-or-temp-table-postgres

Reply:
Changed using temporary tables to using variables.